files/modrinth-exclude-include.json

```
@@ -46,6 +46,7 @@
     "FauxCustomEntityData",
     "feytweaks",
     "figura",
+    "forge-config-screens",
```

It's annoying, but the Modrinth exclusions have to match part of the file name and not the slug. For example, looking at https://modrinth.com/mod/forge-config-screens/version/v8.0.2-1.20.1-Forge, the partial filename would be "ForgeConfigScreens"


```suggestion
    "ForgeConfigScreens",
```

---

Oh I see, thanks for the heads up. I'll fix it right away